### PR TITLE
Update groupdata of Picture-In-Picture API & some other update

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

update groupdata of Picture-In-Picture API - remove non-existed guide, add missing interface and property, remove an event as the interface it belongs to has in the list already

rewrite value section of `ShadowRoot.pictureInPictureElement`

add `PictureInPictureEvent` to API interface list, and fixes in permissions-policy (seems to be conflict with fullscreen?)

### Related issues and pull requests

relates to https://github.com/mdn/translated-content/issues/23557

### CODE REVIEW

File: files/jsondata/GroupData.json [hunk 1]
```diff
@@ -1161,21 +1161,24 @@
     },
     "Picture-in-Picture API": {
       "overview": ["Picture-in-Picture API"],
-      "guides": ["/docs/Web/API/Picture-in-Picture_API/Guide"],
-      "interfaces": ["PictureInPictureWindow"],
+      "guides": [],
+      "interfaces": [
+        "PictureInPictureWindow",
+        "PictureInPictureEvent"
+      ],
```
Comment (skyclouds2001) [inline files/jsondata/GroupData.json:1168]:
```suggestion
      "interfaces": ["PictureInPictureWindow", "PictureInPictureEvent"],
```

File: files/en-us/web/api/picture-in-picture_api/index.md [hunk 2]
```diff
@@ -71,7 +73,7 @@ The [`:picture-in-picture`](/en-US/docs/Web/CSS/:picture-in-picture) [CSS](/en-U
 
 ## Controlling access
 
-The availability of picture-in-picture mode can be controlled using [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy). The fullscreen mode feature is identified by the string `"picture-in-picture"`, with a default allowlist value of `"self"`, meaning that picture-in-picture mode is permitted in top-level document contexts, as well as to nested browsing contexts loaded from the same origin as the top-most document.
+The availability of picture-in-picture mode can be controlled using [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy). The picture-in-picture mode feature is identified by the string `"picture-in-picture"`, with a default allowlist value of `"*"`, meaning that picture-in-picture mode is permitted in top-level document contexts, as well as to nested browsing contexts loaded from the same origin as the top-most document.
```
Comment (bsmth) [inline files/en-us/web/api/picture-in-picture_api/index.md:76]: should `*` be quoted? How does it appear in the header?

Comment (skyclouds2001) [reply files/en-us/web/api/picture-in-picture_api/index.md:76]: A usage is like: `Permissions-Policy: picture-in-picture=*`

Comment (bsmth) [reply files/en-us/web/api/picture-in-picture_api/index.md:76]: I would remove the quotation marks here, "*" is seen a lot in the spec, but this means literal `*` char, as far as I can tell.

Comment (skyclouds2001) [inline files/en-us/web/api/picture-in-picture_api/index.md:76]:
```suggestion
The availability of picture-in-picture mode can be controlled using [Permissions Policy](/en-US/docs/Web/HTTP/Permissions_Policy). The picture-in-picture mode feature is identified by the string `"picture-in-picture"`, with a default allowlist value of `*`, meaning that picture-in-picture mode is permitted in top-level document contexts, as well as to nested browsing contexts loaded from the same origin as the top-most document.
```

**Review by bsmth — APPROVED** [review]
Thank you! 👍🏻

**Pull request merged**